src/Directive.php

```diff
@@ -29,10 +32,27 @@ abstract class Directive
     public const WAKE_LOCK = 'wake-lock';
     public const XR = 'vr';
 
-    public static function isValid(string $directive): bool
+    public static function make(string $directive, string $type = 'default'): DirectiveContract
+    {
+        return match ($type) {
+            'default' => DefaultDirective::directive($directive),
```

Comment:
The `DefaultDirective` class holds the default or 'stable' directives. Moved it to a separate class to keep this class small.